qt/scientific_interfaces/ISISReflectometry/test/RunsTable/RunsTablePresenterDisplayTest.h

```diff
@@ -245,6 +245,26 @@ class RunsTablePresenterDisplayTest : public CxxTest::TestSuite, RunsTablePresen
     verifyAndClearExpectations();
   }
 
+  void testNotifyBatchRowCellChanged() {
+    auto reductionJobs = twoGroupsWithMixedRowsModel();
+    auto presenter = makePresenter(m_view, reductionJobs);
+    auto const rowLocation = location(0, 0);
+    ON_CALL(m_jobs, cellAt(rowLocation, 0)).WillByDefault(Return(Cell("")));
+    EXPECT_CALL(m_mainPresenter, notifyRowContentChanged(_)).Times(1);
```

Comment:
Could these tests check that the correct row/group is being passed in the expected call?

Reply:
I can't really see a way to do this without some heavy refactoring. The updateRowField function makes a call to updateRow which causes the row in reduction jobs to be replaced. This makes it hard to compare the rows